### DISCUSSION TITLE
Should we have a maintenance label?

### DISCUSSION THREAD

**Discussion by nickkaczmarek** [wwt/SwiftCurrent#174] [General]
When reviewing [this PR for fast lane changes](https://github.com/wwt/SwiftCurrent/pull/172) it struck me that this isn't really a `bug` because it's not a bug in our code, but is instead more like a `maintenance` task. For this reason, should we have a `maintenance` label for things like updating fastlane, swiftlint, etc?

**Comment by nickkaczmarek** [comment]
@morganzellers @Tyler-Keith-Thompson Curious what y'all think.

**Comment by morganzellers** [comment]
I think this is a great suggestion and something we should do

**Reply by Tyler-Keith-Thompson** [reply]
I like this from a changelog perspective. As long as we don't get label overload I think this is worth it.

**Comment by nickkaczmarek** [comment]
Alright. I'll set it up.
